polkadot/runtime/common/src/claims.rs

```diff
@@ -1471,6 +1489,12 @@ mod benchmarking {
 	}
 
 	benchmarks! {
+		where_clause { where <T as frame_system::Config>::RuntimeCall: IsSubType<Call<T>> + From<Call<T>>,
+			<T as frame_system::Config>::RuntimeCall: Dispatchable<Info = DispatchInfo> + GetDispatchInfo,
+			<<T as frame_system::Config>::RuntimeCall as Dispatchable>::RuntimeOrigin: AsSystemOriginSigner<T::AccountId> + Clone,
```

Comment:
Actually why do we use `AsSystemOriginSigner` and not simply `OriginTrait::as_signer` ?

Maybe `AsSystemOriginSigner` is not needed and we can just use `OriginTrait::as_signer` everywhere?

Reply:
Ah I think it started out as an abstraction layer to help with getting a ref to a signed origin, but I think we don't really need the abstraction and we implement the trait along with `OriginTrait` anyway in `construct_runtime`. I'll see if there are any barriers to removing it.